### DISCUSSION TITLE
Tutorial

### DISCUSSION THREAD

**Discussion by data-handler** [redom/redom#334] [General]
I’ve been using re:dom for a few months now and like it a lot. One thing I’d like to see in the documentation is a series of tutorials detailing best practices when designing an app with this library. Failing that, more examples of using re:dom in real world scenarios.

**Comment by NetLancer** [comment]
Yeah, i'd love to see Re:DOM lib growing in popularity & use cases, and i believe this is primarily matter of tutorials and/or rich documentation which should be rich in examples. 
Unfortunately i personally don't see many articles or youtube videos on this nice practical tool ..

**Reply by NetLancer** [reply]
Would be great to have tutorials/articles on Re:DOM full stack with say Strapi, FastAPI(python) backend tools, i highly hope we'll see some in foreseeable future :))

**Comment by pakastin** [comment]
Feel free to create PRs:
https://github.com/redom/redom-docs 😎

**Comment by pakastin** [comment]
There's not that much black magic compared to more complex frameworks and view libraries, so it's almost like using vanilla DOM APIs, with just some added helpers.

Having said that, I do have a lot of best practises I use myself, and have thought many times about creating YouTube videos or blog posts about them. I've just been so busy lately.

**Reply by mrwanashraf** [reply]
please let me know if you ever need help, I would be willing to help out as soon as I can.
